resources/vbmc/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM registry.hub.docker.com/library/python:3.7
 
 RUN apt update && \
-    apt install -y libvirt-dev && \
-    pip3 install virtualbmc
+    apt install -y curl libvirt-dev && \
+    curl -Lf -o /tmp/uc.txt https://releases.openstack.org/constraints/upper/master && \
```

Comment:
instead of using curl we can use -c with the url directly

Reply:
Oh, it works? Nice!